Dockerfile.backend

```diff
@@ -6,7 +6,7 @@ ENV LC_ALL=C.UTF-8
 ENV DEBIAN_FRONTEND=noninteractive
 
 RUN apt-get update && \
-    apt-get install -y python3 libhdf5-dev python3-h5py gettext moreutils build-essential libxml2-dev python3-dev python3-pip zlib1g-dev python3-requests python3-aiohttp llvm jq && \
+    apt-get install -y python3 libhdf5-dev python3-h5py gettext moreutils build-essential libxml2-dev python3-dev python3-pip zlib1g-dev python3-requests python3-aiohttp llvm jq libvips && \
```

Comment:
are you sure libvips is needed for the backend. The "backend" container refers to the the server hosting the API, I don't see any API calls or share business logic that uses pylibvips. I even think the `backend/common/utils/spatial.py` could live with in `backend/layers/processing/spatial.py`

Reply:
NVM, this isn't your fault. There is a lot of stuff in the `backend/common/utils` that needs to be moved elsewhere. If you could please move all code that depends on libvips and pylibvips under `backend/layers/processing/`. We should avoid adding unnecessary dependencies to containers if we can avoid it.

Reply:
for some reason I needed it inside the backend container when running unit tests locally - when I pushed the code I had to add it to the processing container in order to get unit tests to pass